catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/SceneListFragment.java

```diff
@@ -54,7 +54,7 @@
 import static org.catrobat.catroid.common.Constants.Z_INDEX_BACKGROUND;
 import static org.catrobat.catroid.common.SharedPreferenceKeys.SHOW_DETAILS_SCENES_PREFERENCE_KEY;
 
-public class SceneListFragment extends RecyclerViewFragment<Scene> implements ProjectLoadTask.ProjectLoadListener {
+public class SceneListFragment extends RecyclerViewFragment<Scene> implements ProjectLoader.ProjectLoadListener {
```

Comment:
I saw that the ProjectLoader was removed from this class as part of a different ticket. Is there any need for the ProjectLoadListener here now?